input/fsh/invariants/organisationsnummer-invariant.fsh

```diff
@@ -1,5 +1,5 @@
 Invariant: organisationsnummer-invariant
-Description: "All identifiers that identifies as samordningsnummer SHALL comply with the specified regex: 
+Description: "All identifiers that identifies as organisationsnummer SHALL comply with the specified regex: 
```

Comment:
Please remove the additional extra new line at the end of this line. Otherwise, This will make a newline character in the generated SD.